app/build.gradle

```diff
@@ -45,9 +65,9 @@ android {
 
     defaultConfig {
         applicationId "info.nightscout.android"
-        minSdkVersion 14
```

Comment:
Is there a need to change the minSdkVersion? I think we still have some folks using 4.0...